.gitignore

```diff
@@ -0,0 +1,72 @@
+node_modules/
```

Comment:
This file feels very auto-generated and untidy. We can clean this up and simplify it :)